less/admin/AdminNav.less

```diff
@@ -86,3 +181,26 @@
     }
   }
 }
+
+.ExtensionListItem-Dot {
```

Comment:
The menu nicely takes care of (if overflow => ellipsis) cases, but with the nice Dot to visually show if an extension is enabled/disabled, it covers the text. It would be nice to have somewhere something like:

```css
.ExtensionNavButton .Button-label {
  display: inline-block;
  max-width: calc(100% - 18px);
  overflow: hidden;
  text-overflow: ellipsis;
}
```